gdx/src/com/badlogic/gdx/graphics/g3d/decals/AlphaTestGroupStrategy.java

```diff
@@ -0,0 +1,151 @@
+package com.badlogic.gdx.graphics.g3d.decals;
+
+import com.badlogic.gdx.*;
+import com.badlogic.gdx.graphics.*;
+import com.badlogic.gdx.graphics.glutils.*;
+import com.badlogic.gdx.utils.*;
+
+/**
+ * <p>Single Group strategy (all Decals same) using Z-buffer to render using screen door transparency to avoid having
+ * to presort the decals. Materials are still sorted.</p>
```

Comment:
presort decals is not an issue, the reason of this class is to avoid overlap artifacts in some situation.